ui/admin/tests/acceptance/storage-buckets/selectors.js

```diff
@@ -3,11 +3,14 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+// START TODO: Remove common selectors from storage-bucket specific selectors
 export const SAVE_BTN = '[type=submit]';
 export const CANCEL_BTN = '.rose-form-actions [type=button]';
 export const EDIT_BTN = '.rose-form-actions [type=button]';
 export const FIELD_NAME = '[name=name]';
 export const FIELD_NAME_VALUE = 'Random name';
+// END TODO: Remove common selectors from storage-bucket specific selectors
```

Comment:
I will remove these in a separate PR. Where I just remove these and update the tests that were using them.